src/db.ts

```diff
@@ -263,7 +263,8 @@ export const getDevice = async (deviceId) =>
     )
   );
 
-export const getAllDevices = async () => {
+// DEPRECATED
```

Comment:
If you use @deprecated, IDEs recognize it and use strikethrough on usages
https://jsdoc.app/tags-deprecated.html